### PR TITLE
feat: add new transformed cube js dashboard

### PR DESCRIPTION
**Pull request by vigilantee**

### CODE REVIEW

**Comment by vigilantee** [comment]
Added a new dashboard ui skin to the powerful cube js dashboard.

**Comment by vigilantee** [comment]
Screenshot 2020-12-15 at 12 08 23 PM
Screenshot 2020-12-15 at 12 08 52 PM

**Review by guptamanish07 — CHANGES_REQUESTED** [review]
I think we don't need config.js

File: bloomstack_core/public/js/bloomstack_core/admin_insights/components/Query.js [hunk 1]
```diff
@@ -396,145 +384,117 @@ const tabItemProductCount = {
...
-    });
-};
+const TabSalesInvoiceTopCustomerByRevenue = {
+    "measures": [
+        "TabSalesInvoice.trueGrandTotal"
+    ],
+    "timeDimensions": [
+        {
+            "dimension": "TabSalesInvoice.postingDate",
+            "dateRange": "This year"
```
Comment (guptamanish07) [inline bloomstack_core/public/js/bloomstack_core/admin_insights/components/Query.js]: Do we removing dynamic feature?

File: bloomstack_core/public/js/bloomstack_core/admin_insights/Home.vue [hunk 2]
```diff
@@ -252,164 +209,99 @@
...
-    },
-
-    tabPaymentEntryWeeklyRevnue(){
-      return Query.tabPaymentEntryWeeklyRevnue;
-    },
-
-    tabPaymentEntryMonthlyRevnue(){
-      return Query.tabPaymentEntryMonthlyRevnue;
-    }
+    return { ...dataObj, startDate, endDate };
```
Comment (guptamanish07) [inline bloomstack_core/public/js/bloomstack_core/admin_insights/Home.vue]: if you do these changes then how we will paas startDate and endDate to a particular Query

File: bloomstack_core/public/js/bloomstack_core/admin_insights/Home.vue [hunk 3]
```diff
@@ -1,45 +1,31 @@
 <template>
   <div class="container-fluid">
-    <div style="display: flex">
-      From
-      <datepicker v-model="startDate" format="dd-MM-yyyy"></datepicker>
-      To
-      <datepicker v-model="endDate" format="dd-MM-yyyy"></datepicker>
-      <select
```
Comment (guptamanish07) [inline bloomstack_core/public/js/bloomstack_core/admin_insights/Home.vue]: Do we don't need this feature?

**Pull request closed without merging**